docs/utilities/parameters.md

```diff
@@ -166,6 +171,86 @@ in order to get data from other regions or use specific credentials.
     }
     ```
 
+## AppConfig
+To retrieve parameters from AppConfig, you can choose to use the [AppConfig Lambda Extension](https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-integration-lambda-extensions.html)
+or the client SDK. 
+
+!!! info
+    See how to create configuration in AppConfig in the [documentation](https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-working.html).
+
+=== "AppConfigProvider"
+
+    ```java hl_lines="5 8-9"
+    import software.amazon.lambda.powertools.parameters.AppConfigProvider;
+    import software.amazon.lambda.powertools.parameters.ParamManager;
+
+    public class AppWithConfig implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+        // Get an instance of the AppConfig Provider
+        AppConfigProvider appConfigProvider = ParamManager.getAppConfigProvider();
+    
+        // Retrieve some configuration
+        // The key must be in form /application/environment/configuration and match your AppConfig setup
+        String value = appConfigProvider.get("/app/prod/config");
+    
+    }
+    ```
+
+=== "AppConfigProvider with a custom client"
+
+    ```java hl_lines="5 8 11"
+    import software.amazon.lambda.powertools.parameters.AppConfigProvider;
+    import software.amazon.lambda.powertools.parameters.ParamManager;
+
+    public class AppWithConfig implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+        AppConfigDataClient client = AppConfigDataClient.builder().region(Region.EU_CENTRAL_1).build();
+        
+        // Get an instance of the AppConfig Provider
+        AppConfigProvider appConfigProvider = ParamManager.getAppConfigProvider(client);
+    
+        // Retrieve a single secret
+        String value = appConfigProvider.get("/app/prod/config");
+    
+    }
+    ```
+
+### Using the AppConfig Extension for Lambda
```

Comment:
This is the part I was referring to @jeromevdl - we don't support this in Python.

That's why I was asking for a RFC to understand the behaviour so it could be assessed for other languages later.

Does that make sense?